src/resource/base64encode.js

```diff
@@ -0,0 +1,3 @@
+exports.image = [
```

Comment:
Why are using a base64 encoded file here?

Reply:
want to upload food images as base65encoded files

Reply:
Why do you want to do that? Also, I sent you a message about the way your are exporting. Please refer to the documents I shared about the es6 documentation for exports.